node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pendulum-node"
-version = "0.1.0"
+version = "1.1.0"
```

Comment:
This is noted. 

We'll just have to manually create a release note and tags of the Node for the time being.
:thumbsup: